### PR TITLE
AL_2019

### PR DESCRIPTION
**Pull request by vivacebelles**
NWChem edits for current QCDB.

### CODE REVIEW

File: qcdb/intf_nwchem/options.py [hunk 1]
```diff
@@ -397,29 +403,66 @@ def load_nwchem_defaults(options):
...
+            default=1.2, 
+            validator=lambda x: float(x), 
+            glossary='Scaling factor for same spin'))
 
     options.add(
         'nwchem',
         RottenOption(
-            keyword='mp2_fos', default=0.3, validator=lambda x: float(x), glossary='Scaling factor for opposite spin'))
+            keyword='mp2__fos', 
+            default=0.3,
```
Comment (loriab) [inline qcdb/intf_nwchem/options.py]: this is 1/3, I think

File: qcdb/intf_nwchem/options.py [hunk 2]
```diff
@@ -600,31 +645,31 @@ def load_nwchem_defaults(options):
     options.add(
         'nwchem',
         RottenOption(
-            keyword='dft_max_ovl',
+            keyword='dft__max__ovl',
```
Comment (loriab) [inline qcdb/intf_nwchem/options.py]: `dft__max_ovl` since `max_ovl` is name of the keyword

File: qcdb/intf_nwchem/options.py [hunk 3]
```diff
@@ -704,66 +750,232 @@ def load_nwchem_defaults(options):
...
-#            'nwchem',
-#            RottenOption(
-#                keyword='ccsd(t)__thresh',
-#                default= 1.0e-6,
-#                validator=parsers.parse_convergence,
-#                glossary= 'Convergence threshold for the iterative part of the calculation.'))
+    options.add(
+            'nwchem',
+            RottenOption(
+                keyword= 'ccsd_pr_t__maxiter',
```
Comment (loriab) [inline qcdb/intf_nwchem/options.py]: I think the maxiter for ccsd(t) is handled in the CCSD block, so these shouldn't be needed. No iterations associated with the (T) step.

File: qcdb/intf_nwchem/options.py (hunk 2 above)
Comment (vivacebelles) [reply qcdb/intf_nwchem/options.py]: Thanks for catching that! I thought I got all of them.

**Pull request closed without merging**